lib/order.rb

```diff
@@ -0,0 +1,28 @@
+# deals with orders + cost
+
+class Order
+
+	def initialize(menu:) #dependency injection - 'other class is injected into this class'
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)